test/test_ftps_server.py

```diff
@@ -0,0 +1,83 @@
+import os
+import time
+from ftplib import FTP
+
+import pytest
+
+from domain.ftps_server import FTPsServer
+
+
+@pytest.fixture
+def ftps_server():
+    return FTPsServer(
+        "0.0.0.0", 21, 50, 5, "../cert/server.pem", "../cert/keyserver.pem", "./ftp_root"
```

Comment:
This will probabily fail if keys are not available. Also I think listening to 0.0.0.0 might be disallowed by github action

Reply:
I think 127.0.0.1 is disallowed as well. Take a look at this: https://github.com/stefanodellosa-personal/WADAS/actions/runs/11590528828/job/32268293354?pr=45